index.html

```diff
@@ -1,523 +1,1175 @@
 <!DOCTYPE html>
 <html lang="zxx">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=yes"
+    />
+    <link rel="stylesheet" href="style.css" />
 
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=yes">
-        <link rel="stylesheet" href="style.css">
+    <!-- Fonts & Icons -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      defer
+      src="https://use.fontawesome.com/releases/v5.13.0/js/all.js"
+      integrity="sha384-ujbKXb9V3HdK7jcWL6kHL1c+2Lj4MR4Gkjl7UtwpSHg/ClpViddK9TI7yU53frPN"
+      crossorigin="anonymous"
+    ></script>
 
-        <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap" rel="stylesheet">
-        <script defer src="https://use.fontawesome.com/releases/v5.13.0/js/all.js" integrity="sha384-ujbKXb9V3HdK7jcWL6kHL1c+2Lj4MR4Gkjl7UtwpSHg/ClpViddK9TI7yU53frPN" crossorigin="anonymous"></script>
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/bootstrap.css" />
 
-        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
-        <link rel="stylesheet" href="css/bootstrap.css">
+    <!--MyCustomCSS-->
+    <link rel="stylesheet" href="css/custom.css" />
 
-        <link rel="stylesheet" href="css/custom.css">
+    <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+  </head>
 
-        <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
-    </head>
+  <body class="m-0 customBody">
+    <header class="clearfix">
+      <div
+        class="p-3 b-0 d-flex flex-wrap align-items-center justify-content-between w-100 customHeader"
+      >
+        <div
+          class="float-left w-25 d-none d-lg-block d-none d-lg-block text-nowrap text-left m-0"
+        >
+          <p>
+            <img
+              class="w-25 p-2"
+              src="https://g.newsweek.com/img/weather/26.png"
+              alt="alt"
+            />
+            18ºC, Angra dos Reis
+          </p>
```

Comment:
- [ ] The navbar still not quite there. Try to pay attention to height, margins, element sizes, and font sizes. (please see how the trending section ins aligning to the left, and the | US - World - etc | should cover most of the width.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88434780-51aa4f00-cdd7-11ea-9b6e-151c32b862b0.png) | ![image](https://user-images.githubusercontent.com/61479648/88434761-47885080-cdd7-11ea-90fc-4df204cfd685.png) |